docs/Getting Started/Installation/index.md

```diff
@@ -38,11 +39,53 @@ You can run the following command to know if you have PostgreSQL and also check
 psql -V
 ```
 
-In order to start creating users and databases, you will have to log in to PostgreSQL. It comes with a `postgres` user that has no password by default. Run the following command to log in as the `postgres` user:
+## PostgreSQL Installation (Ubuntu)
+To install PostgreSQL in Ubuntu, run the following commands from the PostgreSQL documentation:
 ```
+# Create the file repository configuration:
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+
+# Import the repository signing key:
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+# Update the package lists:
+sudo apt-get update
+
+# Install the latest version of PostgreSQL.
+# If you want a specific version, use 'postgresql-12' or similar instead of 'postgresql':
+sudo apt-get -y install postgresql
+```
+
+You can run the following command to know if you have PostgreSQL and also check that you are running the most recent version:
+```
+psql -V
+```
+
+
+## Logging in to PostgreSQL
+In order to start creating users and databases, you will have to log in to PostgreSQL. It comes with a `postgres` user that should have no password by default. Run the following command to log in as the `postgres` user:
+```
+psql -U postgres
```

Comment:
I was unable to get this command working for me (using Ubuntu 18.04.2) but `sudo -u postgres psql` let me access the DB. I'm not sure if it's a problem with needing to use sudo.

Reply:
I tested against Ubuntu 18.04 and 20.04 and was able to get both commands to work.
A key difference between these two commands is that using `psql -U <user>` will simply log in to postgres as <user>. Whereas `sudo -u <user> psql` changes the user that is running the psql command and then attempts to log in to postgres as a user of the same name. While this works for the postgres user because it is created when postgres is installed, it adds an extra step when creating new users because they now have to be added to postgres **and** to the system. If the user doesn't exist on the system, it fails with `sudo: unknown user: <user>`.